req-travis-pip.txt

```diff
@@ -1,3 +1,4 @@
 nose2[coverage-plugin]==0.5.0
 coveralls==1.0
-wheel==0.26.0
+picklable-itertools==0.1.1
+progressbar2==2.7.3
```

Comment:
Nevermind: `req.txt` is for users, whereas `req-travis-{pip,conda}.txt` is for Travis.